### PR TITLE
chore: fix security issues

### PR DESCRIPTION
**Pull request by nduchak**
make audit fix of deps

### CODE REVIEW

**Comment by artem-brazhnikov** [comment]
The CircleCI types check is failing

File: config/ganache.json5 [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   port: 3030,
+  auth: {
```
Comment (artem-brazhnikov) [inline config/ganache.json5:3]: same

File: config/test.json5 [hunk 2]
```diff
@@ -2,6 +2,9 @@
   port: 3030,
   dataDir: 'data-test',
   dbBackUp: { blocks: 10, path:  'db-backups' },
+  auth: {
```
Comment (artem-brazhnikov) [inline config/test.json5:5]: same

**Review by artem-brazhnikov — CHANGES_REQUESTED** [review]

**Review by artem-brazhnikov — APPROVED** [review]

File: config/development.json5 [hunk 3]
```diff
@@ -1,5 +1,8 @@
 {
   port: 3030,
+  auth: {
+    secret: '8a58b86565c23c9ea90'
```
[the remark this answers is not in the document]
Comment (nduchak) [reply config/development.json5]: Done

**Review by julianmrodri — APPROVED** [review]
lgtm!

**Review by artem-brazhnikov — APPROVED** [review]

**Pull request merged**